src/content-helper/dashboard-widget/dashboard-widget.scss

```diff
@@ -0,0 +1,125 @@
+@import "../../blocks/shared/variables";
+@import "../../blocks/shared/functions";
+
+#wp-parsely-dashboard-widget {
+
+	.parsely-spinner-wrapper {
+		display: flex;
+		justify-content: center;
+		margin: to_rem(103px) 0;
+
+		svg {
+			height: 22px;
+			width: 22px;
+		}
+	}
+
+	.parsely-contact-us {
+		margin-top: to_rem(15px) !important;
+	}
+
+	p.parsely-error-hint {
+		color: var(--gray-700);
+	}
+}
+
+#wp-parsely-dashboard-widget .parsely-top-posts-wrapper {
```

Comment:
Yes, I've done this on purpose to avoid too many nested levels that can result in huge CSS parent blocks that are harder to read. This approach of defining major CSS blocks separately has been taken in other files as well (see content-helper.scss). This should not have an impact on the generated CSS file, so I do not see a good reason to change this. However, if your opinion differs let me know.